### PR TITLE
PG-513: Display location of most recent export

### PR DESCRIPTION
**Pull request by phillip-hopper**
Also provide a link to open the directory

Resolves #PG-513

[Review on Reviewable](https://reviewable.io/reviews/sillsdev/glyssen/66)

### CODE REVIEW

File: Glyssen/Dialogs/ExportDlg.cs [hunk 1]
```diff
@@ -159,6 +159,10 @@ private void BtnOk_Click(object sender, EventArgs e)
 			try
 			{
 				m_projectExporter.GenerateFile(filePath, m_selectedFileType);
+
+				// remember the location
+				Settings.Default.LastExportLocation = Path.GetDirectoryName(filePath);
```
Comment (andrew-polk) [inline Glyssen/Dialogs/ExportDlg.cs]: I think this needs to be per project, thus needs to be saved in the project metadata.

**Comment by andrew-polk** [comment]
Also, its just an opinion (and you could check with someone else for theirs), but I think the link should line up horizontally with the text next to the other buttons (rather than the check marks).

File: Glyssen/Dialogs/ExportDlg.cs (hunk 1 above)
Comment (phillip-hopper) [inline Glyssen/Dialogs/ExportDlg.cs]: Done.

**Comment by phillip-hopper** [comment]
Horizontal alignment updated.

**Pull request merged**